wasm_sdk/src/egp_env.rs

```diff
@@ -52,8 +52,8 @@ impl EgpEnv {
     /// Parse current environment variables related to emulated guest process execution envirionment.
     pub fn parse() -> Self {
         let mut env = EgpEnv {
-            vm_data_folder: std::env::var("EVM_DATA_FOLDER").expect("Failed to get data folder"),
-            process_data_folder: std::env::var("EGP_DATA_FOLDER").expect("Failed to get emulated process data folder"),
+            vm_data_folders: Self::parse_folders_to_vec ("EVM_DATA_FOLDER"),
```

Comment:
i wonder if it is your ide setting, every function call '(' has a space ' ' before it.... emmmm